exercises/concept/tim-from-marketing/src/main/java/Badge.java

```diff
@@ -0,0 +1,5 @@
+public class Badge {
+    public static String print(Integer id, String name, String department) {
```

Comment:
In the Java track we try to minimize the use of static methods. Especially in concept exercises I believe there is no other exercise that uses static methods, so that would technically introduce that concept here. 

I would prefer if we could use an instance method instead, even if it doesn't actually interact with the `Badge` class instance.